packages/server/src/auth/dto/auth.dto.ts

```diff
@@ -117,7 +116,7 @@ export class SignupTransformPipe implements PipeTransform {
     const user = new UserSignupDto();
 
     user.projectId = body.projectId.toString();
-    user.username = body.username.toString();
+    // user.username = body.username.toString();
```

Comment:
Won't this make it so that the signup object never has a username (even if one is provided)? Should probably check to see if the username is on the body